### PR TITLE
Update Configure Cluster Resources

### LINKED ISSUE #1465: Reconsider cluster modification docs /reference docs structure
**Affected Document:** https://docs.astronomer.io/astro/category/configure-cluster-resources

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [X] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):** 

Let's have a more clear split between our action-oriented cluster docs and our reference cluster docs, and let's update our reference docs to have more clear language than "resources"

**Additional Notes:**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1465 

@jwitz I attempted to clarify the topic titles. The sub-topic _Request and confirm a cluster change_ is not a task; it's informational. Feel free to implement any changes you think are necessary and merge this PR. Thanks!

### CODE REVIEW

**Comment by jwitz** [comment]
I still believe that _Request and confirm a cluster change_ is a task, with the core action being "Contact Astronomer support". Say you're an admin user trying to figure out how to change the region for one of your clusters. You go to "Manage cluster resource settings", but all you see is reference documentation. It's not clear where to go when you already know what cluster change you want and whether it's supported. 

A doc should serve this persona, and I think turning "Supported cluster modifications" into "Request a cluster change" would do just that. We could cut a lot of the content from the opening section in "Supported cluster modifications" so that you can see the call to contact Astronomer support without scrolling. Most of that information can be removed by instead linking out to the new category menu you created. Let me know if you want me to commit some of these ideas to the PR to see them live on the preview site!

**Comment by kwcanuck** [comment]
@jwitz as stated above, "feel free to implement any changes you think are necessary and merge this PR". No need to discuss it further.

File: sidebarsAstro.js [hunk 1]
```diff
@@ -103,12 +103,12 @@ module.exports = {
             "create-cluster",
             {
               type: "category",
-              label: "Configure cluster resources",
+              label: "Manage cluster resource settings",
```
Comment (jwitz) [inline sidebarsAstro.js:106]:
```suggestion
              label: "Manage cluster settings",
```

**Comment by jwitz** [comment]
I tried keeping the reference docs in two different sidebar menus but was getting weird behavior when navigating though the sidebar. Electing to keep the reference docs in the Reference tab for now.

**Pull request merged**